exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/internal/PrometheusMetricReaderProvider.java

```diff
@@ -32,6 +39,20 @@ public MetricReader createMetricReader(ConfigProperties config) {
     if (host != null) {
       prometheusBuilder.setHost(host);
     }
+    String defaultHistogramAggregation =
+        config.getString("otel.exporter.prometheus.metrics.default.histogram.aggregation");
```

Comment:
```suggestion
        config.getString("otel.java.experimental.exporter.prometheus.metrics.default.histogram.aggregation");
```

Reply:
Sure, updated it to experimental property